core/tests/protractor_desktop/explorationTranslationTab.js

```diff
@@ -48,253 +48,272 @@ describe('Exploration translation and voiceover tab', function() {
-    explorationEditorPage.navigateToTranslationTab();
-    explorationEditorTranslationTab.changeLanguage('Hindi');
-    explorationEditorTranslationTab.exitTutorial();
-    explorationEditorTranslationTab.expectToBeInVoiceoverMode();
-    users.logout();
+  it('should have voiceover as a default mode', async function() {
+    await users.login('voiceArtist@translationTab.com');
+    await creatorDashboardPage.get();
+    await creatorDashboardPage.editExploration('Test Exploration');
+    await explorationEditorMainTab.exitTutorial();
+    await explorationEditorPage.navigateToTranslationTab();
+    await explorationEditorTranslationTab.changeLanguage('Hindi');
+    await explorationEditorTranslationTab.exitTutorial();
+    await explorationEditorTranslationTab.expectToBeInVoiceoverMode();
+    await users.logout();
   });
 
   it('should have all the state contents for voiceover in exploration language',
-    function() {
-      users.login('voiceArtist@translationTab.com');
-      creatorDashboardPage.get();
-      creatorDashboardPage.editExploration('Test Exploration');
-      explorationEditorPage.navigateToTranslationTab();
-      explorationEditorTranslationTab.changeLanguage('English');
-      explorationEditorTranslationTab.expectContentTabContentToMatch(
+    async function() {
+      await users.login('voiceArtist@translationTab.com');
+      await creatorDashboardPage.get();
+      await creatorDashboardPage.editExploration('Test Exploration');
+      await explorationEditorMainTab.exitTutorial();
+      await explorationEditorPage.navigateToTranslationTab();
+      await explorationEditorTranslationTab.changeLanguage('English');
+      await explorationEditorTranslationTab.expectContentTabContentToMatch(
         'This is first card.');
-      explorationEditorTranslationTab.expectFeedbackTabContentsToMatch(
+      await explorationEditorTranslationTab.expectFeedbackTabContentsToMatch(
         ['This is feedback1.', 'This is default_outcome.']);
-      explorationEditorTranslationTab.expectSolutionTabContentToMatch(
+      await explorationEditorTranslationTab.expectSolutionTabContentToMatch(
         'This is solution.');
-      explorationEditorTranslationTab.expectHintsTabContentsToMatch(
+      await explorationEditorTranslationTab.expectHintsTabContentsToMatch(
         ['This is hint1.', 'This is hint2.']);
-      users.logout();
+      await users.logout();
     });
 
-  it('should contain accessibility elements', function() {
-    users.login('voiceArtist@translationTab.com');
-    creatorDashboardPage.get();
-    creatorDashboardPage.editExploration('Test Exploration');
-    explorationEditorPage.navigateToTranslationTab();
+  it('should contain accessibility elements', async function() {
+    await users.login('voiceArtist@translationTab.com');
+    await creatorDashboardPage.get();
+    await creatorDashboardPage.editExploration('Test Exploration');
+    await explorationEditorMainTab.exitTutorial();
```

Comment:
When I ran the tests individually with "fit", this test seemed to fail because the tutorial showed up (it only shows up the first time the exploration is edited), so to make the tests independent of the changes before it I've added the exitTutorial in all the it blocks (exitTutorial simply dismisses the modal if present).